HopsanGUI/GUIObjects/GUIModelObject.cpp

```diff
@@ -1344,6 +1346,13 @@ QAction *ModelObject::buildBaseContextMenu(QMenu &rMenu, QGraphicsSceneContextMe
     {
         openPropertiesDialog();
     }
+    else if(selectedAction == sourceCodeAction)
+    {
+        auto& appearance = gpLibraryHandler->getModelObjectAppearancePtr(mModelObjectAppearance.getTypeName());
```

Comment:
Reference again, see other comment
also  below, using basePath without /, not sure if it is actually a problem